docs/kind-webserver.md

```diff
@@ -0,0 +1,115 @@
+## Webserver definition
+
+The webserver builder can be used to create a Docker image of a webserver. It
+supports two different ways to invoke it:
+
+1. alone, with its own zbuildfile (e.g. if you want to only serve an API with
+no assets) ;
+2. integrated within another builder (e.g. to add assets built by another
+zbuildfile) ;
+
+## Syntax
+
+zbuildfiles with webserver types have following structure:
+
+```yaml
+kind: webserver
+
+webserver: <string>
+system_packages: <map[string]string>
+config_file: <string>
+healthcheck: <bool>
```

Comment:
isn't it default to `true` ? maybe the default value should be specified in the doc

Reply:
It's written below, in the section dedicated to `healthcheck`. Do you think it's really useful to repeat it in this section? And what do you think of this notation?

```yaml
healthcheck: <bool> (default: true)
```

Reply:
> It's written below

oh yes, right :sweat_smile: 

> And what do you think of this notation ?

:+1: :ok_hand: